guides/doc-Planning_for_Project/master.adoc

```diff
@@ -15,7 +15,7 @@ ifdef::satellite[]
 include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
 endif::[]
 
-include::common/modules/con_foreman-overview-and-concepts.adoc[]
+include::common/modules/con_foreman-overview-and-concepts.adoc[leveloffset=+1]
```

Comment:
This is an issue I also found in https://github.com/theforeman/foreman-documentation/pull/3634

If I remember correctly, **two** are missing a leveloffset of 1.

Reply:
This is the only one that was causing failures here. I don't want to overcommit to start fixing all of them :) I just need this to make the ribbon previews working. If you fix this as part of #3634, that's IMO preferable.

Reply:
I think that decision was handed to you in #3634. If adding the leveloffset is good, then I'll do that ASAP.

Reply:
Ack, adding the leveloffset sounds good @maximiliankolb